Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/D7YIGPositionCalibrationTest.py

```diff
@@ -0,0 +1,44 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+from mantid.simpleapi import config, mtd, D7YIGPositionCalibration, Load, LoadILLPolarizedDiffraction
+from mantid.api import ITableWorkspace
+import os.path
+from os import path
+
+
+class D7YIGPositionCalibrationTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        config.appendDataSearchSubDir('ILL/D7/')
+
+    def setUp(self):
+        Load('402652_403041.nxs', OutputWorkspace='shortWavelengthScan')
+
+    def tearDown(self):
+        mtd.clear()
+        if path.exists('test_shortWavelength.xml'):
```

Comment:
The test fails for me: 
```
unclosed file <_io.TextIOWrapper name='/Users/vardanyan/Desktop/test_shortWavelength.xml' mode='w' encoding='UTF-8'>
F
======================================================================
FAIL: test_shortWavelength (D7YIGPositionCalibrationTest.D7YIGPositionCalibrationTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/vardanyan/mantid/Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/D7YIGPositionCalibrationTest.py", line 39, in test_shortWavelength
    self.assertTrue(path.exists('test_shortWavelength.xml'))
AssertionError: False is not true
```

Reply:
I cannot reproduce the error on Linux and Jenkins tests also do not raise any issues, should we take a look at this in person?

Reply:
This still fails, try to use absolute paths with the temp directory from the `os`.